tools/ncnnoptimize.cpp

```diff
@@ -69,6 +70,29 @@
 #include "layer/yolodetectionoutput.h"
 #include "layer/yolov3detectionoutput.h"
 
+#if (defined __aarch64__) && (defined ANDROID)
```

Comment:
it breaks the compilation for armv7 and other os

Reply:
nooop, It has been compiled on Raspberry Pi.